polaris-react/src/components/EmptyState/EmptyState.scss

```diff
@@ -1,134 +1,8 @@
 @import '../../styles/common';
 
-$within-page-detail-width: 336px;
-$detail-width: 400px;
-
-$stacking-order: (
-  illustration: 0,
-  details: 10,
-);
-
-$left-offset: 35%;
-$right-offset: 10%;
-
-// Multiply by 2 to expand the image size to align with design requirements
-$illustration-left-margin: $left-offset * -2;
-
-// Shift the illustration to the left to overlap with Details
-$illustration-width: calc(100% + #{$left-offset} * 2);
-
-// Respect the right offset on mobile so the illustration shows the correct
-// crop zone
-$illustration-width-cropped: calc(
-  #{$illustration-width} + #{$right-offset} * 2
-);
-
-$centered-illustration-width: 226px;
-
-.EmptyState {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  width: 100%;
-  margin: 0 auto;
-  padding: var(--p-space-5) 0;
-  padding-top: var(--p-space-5);
-  padding-bottom: calc(3 * var(--p-space-5));
-  max-width: $page-max-width;
-}
-
-.Section {
-  display: flex;
-  flex-direction: column-reverse;
-  flex: 1 1 auto;
-  width: 100%;
-  align-items: center;
-  justify-content: center;
-
-  @media #{$p-breakpoints-md-up} {
-    left: var(--p-space-5);
-  }
-}
-
-.Details {
-  display: flex;
-  text-align: center;
-  flex-direction: column;
-  align-items: center;
-
-  @media #{$p-breakpoints-sm-up} {
-    max-width: $detail-width;
-  }
-}
-
-.Image {
-  margin: 0;
-  width: initial;
-}
-
-.ImageContainer,
-.DetailsContainer {
-  flex: 1 1 auto;
-  padding: 0;
-  margin: 0;
-
-  @media #{$p-breakpoints-md-up} {
-    flex-basis: 50%;
-  }
-}
-
-.withinContentContainer {
```

Comment:
As far as i can tell this class should always be present since our `EmptyState` components should always be the child of a content container ex. `Card` `Modal` etc. I applied these styles by default to the `Box`es in `EmptyState` but now I'm wondering if I should have conditionally applied these styles instead just in case `EmptyState` is not used in a container parent 🤔

Reply:
went back and addressed this in the component code